nix/modules/default-crates.nix

```diff
@@ -42,7 +43,7 @@
           }
         )
         { }
-        (lib.fileset.toList (globset.lib.globs src cargoToml.workspace.members))
+        (findCrates cargoToml.workspace.members src)
```

Comment:
Hmm, I think the crate-parser should return a set with this function in it. So, `crate-parser.findCrates`.